<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AllergensEditor</class>
 <widget class="QDialog" name="AllergensEditor">
  <property name="windowTitle">
   <string>Editor de alérgenos</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item><widget class="QListWidget" name="lstAllergens"/></item>
   <item>
    <layout class="QHBoxLayout" name="layoutButtons">
     <item><widget class="QPushButton" name="btnAdd"><property name="text"><string>Añadir</string></property></widget></item>
     <item><widget class="QPushButton" name="btnRename"><property name="text"><string>Renombrar</string></property></widget></item>
     <item><widget class="QPushButton" name="btnRemove"><property name="text"><string>Eliminar</string></property></widget></item>
     <item>
      <spacer name="spacerButtons">
       <property name="orientation"><enum>Qt::Horizontal</enum></property>
       <property name="sizeHint" stdset="0"><size><width>40</width><height>20</height></size></property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok|QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
